lib/internal/process/report.js

```diff
@@ -60,6 +60,13 @@ const report = {
     validateBoolean(b, 'compact');
     nr.setCompact(b);
   },
+  get disableNetwork() {
+    return nr.getDisableNetwork();
+  },
+  set disableNetwork(b) {
+    validateBoolean(b, 'disableNetwork');
+    nr.setDisableNetwork(b);
```

Comment:
Nit: setDisable wording is extremely confusing for non-native speakers

```suggestion
    nr.setNetworkDisabled(b);
```